### ISSUE TITLE
New API Keys do NOT funciton within OOE, causes Authentication Error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [EVE-SECURE/ooe#39]
```
What steps will reproduce the problem?
1. Trying to use the new API Keys (CAKs - custom API Key)

What is the expected output? What do you see instead?
Functionality :p

What version of the product are you using? On what operating system?
Latest download (not SVN)

Please provide any additional information below.
Utelization of the new CAKs / API Keys will allow for more functionality, would 
also give you an excuse to do whatever cleanup of the code you feel is needed 
for the API system :)

Would be awsome if you started work on the new KEY system :)

Thank you for such an awsome project!!
```

Original issue reported on code.google.com by `ArcA...@gmail.com` on 31 Oct 2011 at 9:38
- Blocked on: #36
- Merged into: #36
